opam

```diff
@@ -0,0 +1,30 @@
+opam-version: "1.2"
+maintainer: "dave.scott@eu.citrix.com"
```

Comment:
I think we should also update this field to `maintainer: "xen-api@lists.xen.org"`.